tests/materials/processing/models/test_admixture.py

```diff
@@ -0,0 +1,34 @@
+from slamd.materials.processing.models.additional_property import AdditionalProperty
+from slamd.materials.processing.models.admixture import Admixture
+from slamd.materials.processing.models.material import Costs
+
+
+def test_admixture_constructor_sets_default_values():
+    admixture = Admixture()
+
+    assert admixture.name == ''
+    assert admixture.type == ''
+    assert admixture.costs == None
```

Comment:
Comparison with None is usually done with "is" keyword, i.e. for example "admixture.costs is None" instead of admixture.costs == None. Similarly this should be adjusted in the other tests for our models